JSDemos/Demos/Scheduler/RecurringAppointments/description.md

```diff
@@ -1 +1,13 @@
-This example demonstrates how to&nbsp;define recurring appointments. You can specify the appointment repetition interval programmatically or&nbsp;in&nbsp;the UI. In&nbsp;the former case, the appointment object should contain the **recurrenceRule** field. In&nbsp;the latter case, toggle the **Repeat** switcher on&nbsp;the appointment details form to&nbsp;mark the appointment as&nbsp;recurring and specify the required fields.
\ No newline at end of file
+A recurring appointment repeats at a specified interval. In the view, recurring appointments are indicated by a circular arrow glyph.
+
+To create a recurring appointment, do the following:
+
+1. Locate or create an appointment object in the [dataSource](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#dataSource). 
+
+2. Specify the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) property to configure the appointment frequency. Set this property according to the <a href="http://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification. A single data object with **recurrenceRule** creates an appointment series.
+
+1. Optionally, use the [recurrenceException](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceException) property to specify the start date and time of appointments to exclude from the series.
```

Comment:
I cannot say I understood what you meant. What exactly can do with this property and to what end? Should I list there specific date/time values that would match those of potentially excluded appointments? This sentence is not clear.

Reply:
Changed it to "Optionally, use the recurrenceException property to specify the start date and time of **_those appointments that you want to exclude from the series_**."

Reply:
This version looks great. Thank you.